web-services/client/src/main/java/datawave/webservice/query/metric/BaseQueryMetricListResponse.java

```diff
@@ -59,6 +61,14 @@ public void setAdministratorMode(boolean administratorMode) {
         this.administratorMode = administratorMode;
     }
     
+    public boolean isGeoQuery() {
```

Comment:
Since we are changing the response, do we need to start versioning such that we can support a v1 and v2 response?  @brianloss What do you think?

Reply:
The XML and JSON responses are the same.  The HTML response is the only one that changed, FWIW.

Reply:
And the HTML response is ONLY changed if the query contains a geo term...  Again, just providing more food for thought.

Reply:
Yeah, since this is not part of the serialized response, there's no need to version anything.